gradle/libs.versions.toml

```diff
@@ -17,7 +17,7 @@ composeCompiler = "1.5.0"
 detektGradlePlugin = "1.23.0"
 espresso = "3.5.1"
 glance = "1.0.0-alpha05"
-gradle = "8.0.2"
+gradle = "8.1.1"
```

Comment:
This automatic PR just highlights this shouldn't be here, we had another one called agp, I'll look into the diff and consolidate.